etc/schema.json

```diff
@@ -172,19 +172,47 @@
             }
         },
         "app_proto": {
-            "type": "string"
+            "type": "string",
+            "description": "Application layer protocol of the flow",
+            "suricata": {
+                "keywords": [
+                    "app-layer-protocol"
+                ]
+            }
         },
         "app_proto_expected": {
-            "type": "string"
+            "type": "string",
+            "description": "In case of a protocol change to a specific protocol, and this specific protocol was not recognised, this field will have the value of the expected protocol",
+            "suricata": {
+                "keywords": []
```

Comment:
Considering what you're proposing in [this PR](https://github.com/OISF/suricata/pull/13486), should `"keywords" []` be left here, or should we have another version without this? (Or wait for the other one to be merged, and use that format?)